service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/CaseManagementOrderIssuedEventHandler.java

```diff
@@ -87,18 +88,17 @@ public void notifyLocalAuthority(final CaseManagementOrderIssuedEvent event) {
     public void notifyCafcass(final CaseManagementOrderIssuedEvent event) {
         CaseData caseData = event.getCaseData();
 
-        final Optional<Cafcass> recipientIsWelsh =
+        if (CafcassHelper.isNotifyingCafcassWelsh(caseData, cafcassLookupConfiguration)) {
+            final Optional<Cafcass> recipientIsWelsh =
                 cafcassLookupConfiguration.getCafcassWelsh(caseData.getCaseLocalAuthority());
-
-        if (recipientIsWelsh.isPresent()) {
             HearingOrder issuedCmo = event.getCmo();
 
             final IssuedCMOTemplate cafcassParameters = contentProvider.buildCMOIssuedNotificationParameters(
                     caseData, issuedCmo, DIGITAL_SERVICE);
 
             notificationService.sendEmail(
                     CMO_ORDER_ISSUED_NOTIFICATION_TEMPLATE,
-                    recipientIsWelsh.get().getEmail(),
+                    recipientIsWelsh.orElseThrow().getEmail(),
```

Comment:
We should probably add a specific error message or exception you want to throw here